vimiv/api/settings.py

```diff
@@ -493,9 +493,19 @@ class title:  # pylint: disable=invalid-name
 class metadata:  # pylint: disable=invalid-name
     """Namespace for metadata related settings."""
 
+    # Default sets
+    defaults = [
+        "Make, Model, DateTime, ExposureTime, FNumber, IsoSpeedRatings, FocalLength, LensMake, LensModel, PixelXDimension', 'PixelYDimension, ExposureBiasValue",
```

Comment:
Feel free to add a `# pylint: disable=line-too-long,useless-suppression` comment here. I suspect that we are hit by https://github.com/PyCQA/pylint/issues/3368 here, thus the additional `useless-suppression`. I don't think splitting such a set into multiple lines makes much sense.